democracy_club/settings/zappa.py

```diff
@@ -83,3 +83,8 @@
 GOCARDLESS_MERCHANT_ID = os.environ.get('GOCARDLESS_MERCHANT_ID')
 
 SENDGRID_API_KEY = os.environ.get('SENDGRID_API_KEY')
+
+if ZAPPA_STAGE == "dev":
+    AWS_CLOUD_FRONT_ID = "E19EM1B0PXPYCH"
+else:
+    AWS_CLOUD_FRONT_ID = "E3E7Y3O7VH1EMS"
```

Comment:
This probably isn't hugely problematic if all we're doing is invalidating cache, but as a general point, it seems like it would be safer to express this the opposite way round.

i.e: instead of
* "if we're not on dev, assume the production environment" it should actually be
* "if we're not in production, assume the dev environment"

it seems like if we add more environments, or start using this for stuff other than cache invalidation then writing it that way round would be less likely to introduce risk

Reply:
Good call, changed (`E3E7Y3O7VH1EMS` is prod)